Add kubernetes.dev mirroring guidelines (#45168)
kubernetes/website: content/en/docs/contribute/new-content/blogs-case-studies.md

      - **Tutorials** that only apply to specific releases or versions and not all future versions
      - References to pre-GA APIs or features

> ### Mirroring from the Kubernetes Dev blog

Reviewer: ```suggestion
### Mirroring from the Kubernetes Contributor Blog
```

> To mirror a blog post from the [kubernetes.dev contributor blog](https://www.kubernetes.dev/blog/), follow these guidelines:

Reviewer: ```suggestion
To mirror a blog post from the [Kubernetes contributor blog](https://www.kubernetes.dev/blog/), follow these guidelines:
```

- Keep the blog content the same. If there are changes, they should be made to the original article first, and then to the mirrored article.
- The mirrored blog should have a `canonicalUrl`, that is, essentially the url of the original blog after it has been published.

> - Blogs in kubernetes.dev have their authors mentioned in the YAML header, while kubernetes.io blogs mention authors in the blog content itself. This should be changed when mirroring the content.

Reviewer: ```suggestion
- [Kubernetes contributor blogs](https://kubernetes.dev/blog) have their authors mentioned in the YAML header, while the Kubernetes blog posts mention authors in the blog content itself. This should be changed when mirroring the content.
```
